package.json

```diff
@@ -53,8 +53,8 @@
     "depcheck": "lerna exec 'npx depcheck --skip-missing --ignores nan,agent-base,@types/*' --stream"
   },
   "devDependencies": {
-    "@apollo/federation": "^0.38.1",
-    "@apollo/gateway": "^0.50.2",
```

Comment:
Q: Should `@apollo/gateway` be documented on our public docs? If yes, why? If no, why?

Reply:
I am unable to locate any documentation for apollo-gateway in the public documentation. It seems that the majority of the essential tasks related to tracing Apollo Federation gateways (implemented by @apollo/gateway) are handled by our standard GraphQL tracing. The intricate workings of apollo-gateway, which underpins Apollo Federation, might be the reason it is not included in the documentation—this is my speculative assumption.

I propose that we consider adding information about apollo-gateway and subgraph under RPC.

Reply:
> I propose that we consider adding information about apollo-gateway

Yes I think we need to document https://www.npmjs.com/package/@apollo/gateway

Reply:
I will create a PR for public doc update.